src/GoodsList.tsx

```diff
@@ -7,9 +7,13 @@ type Props = {
 
 export const GoodsList: React.FC<Props> = ({ goods }) => (
   <ul>
```

Comment:
use  React.memo() to optimise your code  - checklist p.1.

Reply:
wrapping it with `memo()` would be a good optimization